content/post/057-ipfs-desktop-0-9.md

```diff
@@ -0,0 +1,65 @@
+
+![Evolution of IPFS Desktop](/057-ipfs-desktop-0-9/evolution.jpg)
+
+Regarding the application's name, it went from "ipfs-electron" to "IPFS Native Application", then to "IPFS Station", it being a much more fancier name. However, we changed it again to "IPFS Desktop" and that's where we are right now!
+
+## IPFS Desktop 0.9.0
+
+The newest version of IPFS Desktop, 0.9.2 - yes, we already released two patch versions! - brings more stability and some new experimental features. Although it is not a completely revamped application, such as what happened with 0.7.0, I believe this version deserves an overview of the current (new) features and plans for the future.
+
+### Browse the wild...
+
+With the recent update to the Web UI, we are now capable of navigating through the whole IPFS system. Previously, you could only access your 'Files' (the contents of the [Mutable File System](https://docs.ipfs.io/guides/concepts/mfs/) for those who enjoy technical terms). Now, you can navigate not only to any `/ipfs/` or `/ipns/` path, but also any content [pinned](https://docs.ipfs.io/guides/concepts/pinning/) to your IPFS node!
+
+![Browing /ipns/ipns.io](/057-ipfs-desktop-0-9/browse-wild.png)
+
+### Use npm on ipfs
+
+In addition, we are introducing a new experiments section on the 'Settings' page which will allow you to enable new experimental features as they come in. To celebrate this new section, we are adding [npm on ipfs](https://github.com/ipfs-shipyard/npm-on-ipfs) to it! This is a project that allows you to use the `ipfs-npm` command to install your Node.js modules through the IPFS network!
+
+![Experiments section](/057-ipfs-desktop-0-9/npm-ipfs.png)
+
+**Please note** that this feature is intended for those who have Node.js installed on their system. Moreover, the activation will only work if your setup does not require additional permissions for executing `npm install -g`.
+
+### Add `ipfs` command line tools to your system
+
+Adding `ipfs` command line tools to your system was actually introduced in 0.8.0, but it's always worth noting that you are just a click away of having the `ipfs` command available from your command line! This way, you don't have to worry about configuring your `PATH` variable or doing other confusing stuff to start using all IPFS capabilities. Just go to Settings and enable "IPFS command line tools"!
+
+![IPFS command line tools](/057-ipfs-desktop-0-9/ipfs-cmd.png)
+
+### And more...
+
+Other notable features from previous releases include:
+
+- **Handle `ipfs://`, `ipns://` and `dweb:` links**. If you have IPFS Desktop installed and you click on [ipns://ipfs.io](ipns://ipfs.io), that request will go through be redirected to your own gateway via the app, or to the public gateway if yours is not online.
```

Comment:
Just for extreme clarity, maybe add "... click on ipns://ipfs.io in your browser or any other application,..."